spec/status_spec.rb

```diff
@@ -1,51 +1,49 @@
-require 'spec_helper'
-
 describe CXML::Status do
-  describe 'instance' do
-    it { should respond_to :code }
-    it { should respond_to :text }
-    it { should respond_to :xml_lang }
-    it { should respond_to :success? }
-    it { should respond_to :failure? }
+  describe "instance" do
+    it { is_expected.to respond_to :code }
+    it { is_expected.to respond_to :text }
+    it { is_expected.to respond_to :xml_lang }
+    it { is_expected.to respond_to :success? }
+    it { is_expected.to respond_to :failure? }
   end
 
-  describe '#initialize' do
-    it 'assigns attributes from string' do
-      str = '<Status xml:lang="en-US" code="200" text="OK"></Status>'
+  describe "#initialize" do
+    it "assigns attributes from string" do
+      str = "<Status xml:lang='en-US' code='200' text='OK'></Status>"
```

Comment:
mostly used in XML is double quotas, I think it is better to leave `"` for content.